feature-apkfy/src/main/java/cm/aptoide/pt/feature_apkfy/presentation/ApkfyViewModel.kt

```diff
@@ -42,6 +42,7 @@ class ApkfyViewModel @Inject constructor(
         apkfyManager.getApkfy()
           ?.takeIf { it.packageName != context.packageName }
           ?.takeIf { it.appId != null || it.packageName != null }
+          ?.takeIf { it.utmSource != "AG" && it.utmSource != "AG_dev" }
```

Comment:
Is this change in the scope of the ticket? 🤔

Reply:
Yes, since the logic was changed, if the user installs an app through AG the mmp download call will be done and we will open a bottom sheet with the app the user just installed.